docs/model-dev-guide/api-guides/apis-howto/deepspeed/autotuning.rst

```diff
@@ -4,6 +4,12 @@
  DeepSpeed Autotune: User Guide
 ################################
 
+.. important::
+
+   **Deprecation Notice**: DeepSpeed Autotune has been deprecated. This feature will be removed in a
+   future release. Please see :ref:`Search Methods <_search-methods>` for details on supported
+   preset searchers.
+
```

Comment:
.. important::

   **Deprecation Notice**: DeepSpeed Autotune is deprecated and will be removed in an upcoming release. 
   For information on supported search methods, please visit to :ref:`search-methods`.

Reply:
@azhou-determined i don't think the ref link renders with that extra "_" in front?

Reply:
does it build locally?

Reply:
i think all instances need to be changed